webpack.config.ts

```diff
@@ -59,7 +59,11 @@ export function getClientConfig({
         // local css
         {
           test: /\.css$/,
-          exclude: [path.resolve(__dirname, 'node_modules')],
+          exclude: [
+            __dirname.endsWith(path.sep + 'dist') // __dirname is `[..]/dist` when built server is run in dev
```

Comment:
I don't understand. My understanding is that `__dirname` is the directory the current file is in. This file is top-level and doesn't move?

Reply:
After updating `webpack.config.ts` like so,

<img src="https://user-images.githubusercontent.com/5924865/104536057-6fac8a00-566b-11eb-9c11-ffd6dced7d8c.png" alt="Screenshot of updated webpack config file" width="600px">

Running `yarn dev`:

```
yarn run v1.22.5
$ yarn start:virtual
$ ts-node src/server/dev-server.ts --virtual-robots
[...]
Starting server
{ __dirname: '/home/jpaye/horus/nusight2/dist' }
```

Running `yarn build`:

```
yarn run v1.22.5
$ ts-node build_scripts/build.ts --production
{ __dirname: '/home/jpaye/horus/nusight2' }
```

Running `yarn storybook`:

```
yarn run v1.22.5
$ node conf/storybook/dev.js
[...]
{ __dirname: '/home/jpaye/horus/nusight2' }
```

---

I think the issue is that `webpack.config.ts` gets built into `dist/` when the server is built during dev, then run from there when the client is built.